openedx/stanford/lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -72,6 +72,10 @@ def student_responses(course):
                 elif 'student_answer' in state_dict:
                     raw_answer = state_dict['student_answer']
                     pretty_answers = u"{answer}".format(answer=raw_answer)
+                elif 'attempted' in state_dict and 'completed' in state_dict: # problem-builder
+                    pretty_answers = "{answer}".format(
+                        answer=state_dict,
+                    )
                 else:
                     raw_answers = {}
                     pretty_answers = None
```

Comment:
Interestingly, why don't we at least print the `state_dict` as the fail-safe?

This looks like we don't output anything if we enter this block, right?

Instead of adding your new `elif` check, can't we just use that value of `pretty_answers` in the `else` block, making that the default output?

Reply:
Given that it's not possible to do an analysis of the data, due to size, I'd rather leave as is.